spec/unit/knife/windows_cert_generate_spec.rb

```diff
@@ -0,0 +1,89 @@
+#
+# Author:: Mukta Aphale <mukta.aphale@clogeny.com>
+# Copyright:: Copyright (c) 2014-2016 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "spec_helper"
+require "chef/knife/windows_cert_generate"
+
+describe Chef::Knife::WindowsCertGenerate do
+  before(:all) do
+    @certgen = Chef::Knife::WindowsCertGenerate.new(["-H", "something.mydomain.com"])
+  end
```

Comment:
```suggestion
  let(:certgen) do
    Chef::Knife::WindowsCertGenerate.new(["-H", "something.mydomain.com"])
  end
```

If these tests depend on having the same object this might break...